frontend/src/components/Header/Header.tsx

```diff
@@ -806,20 +747,9 @@ const BillingBanner: React.FC<{ integrated: boolean }> = ({ integrated }) => {
 				[styles.error]: hasExceededSessionsForMonth,
 			})}
 		>
-			<div className={clsx(styles.trialTimeText)}>
-				{bannerMessage}
-				{!canExtend && (
-					<>
-						{' '}
-						Upgrade{' '}
-						<Link
-							className={styles.trialLink}
-							to={`/w/${data?.workspace_for_project?.id}/current-plan`}
-						>
-							here!
-						</Link>
-					</>
-				)}
+			<div className={styles.trialTimeText}>
+				{bannerMessage} Upgrade{' '}
+				<Link to={`/w/${currentWorkspace?.id}/billing`}>here</Link>.
```

Comment:
@ccschmitz just merging your changes into my own branch and caught this - is this `/billing` route valid?

Reply:
@mayberryzane shoot - it's not valid! Should be `/current-plan`. I saw you just updated in your PR - thank you!